docs/ecosystem/ecosystem.md

```diff
@@ -0,0 +1,35 @@
+---
+id: ecosystem
+title:  "Ecosystem"
+---
+
+## ZIO Compatible Libraries
+- [Cats](https://github.com/typelevel/cats), [Cats MTL](https://github.com/typelevel/cats-mtl) and [Cats Effect](https://github.com/typelevel/cats-effect)
+- [Scalaz](https://github.com/scalaz/scalaz)
+- [Monix](https://github.com/monix/monix)
+- [Doobie](https://github.com/tpolecat/doobie)
+- [Http4s](https://github.com/http4s/http4s)
+- [FS2](https://github.com/functional-streams-for-scala/fs2)
+- [Circe](https://github.com/circe/circe)
+- [Reactive Streams](https://github.com/reactive-streams/reactive-streams-jvm)
+- [Twitter Futures](https://github.com/twitter/util)
+
+## Official ZIO Libraries
+
+- [ZIO Actors](https://github.com/zio/zio-actors): A high-performance, purely-functional library for building, composing, and supervising typed actors based on ZIO
+- [ZIO Kafka](https://github.com/zio/zio-kafka): A Kafka client for ZIO and ZIO Streams
+- [ZIO Keeper](https://github.com/zio/zio-keeper): A functional library for consistent replication of metadata across dynamic clusters
+- [ZIO NIO](https://github.com/scalaz/scalaz-nio): A performant, purely-functional, low-level, and unopinionated wrapper around Java NIO functionality
+- [ZIO SQS](https://github.com/zio/zio-sqs): A ZIO-powered client for AWS SQS
+
+## Libraries with direct ZIO Support
+
+- [cakeless](https://github.com/itkpi/cakeless): Better reader monad for deeply-nested cakes
+- [distage](https://github.com/7mind/izumi): Staged, transparent and debuggable runtime & compile-time Dependency Injection Framework
+- [logstage](https://github.com/7mind/izumi): Automatic structural logs from Scala string interpolations
+- [idealingua](https://github.com/7mind/izumi): API Definition, Data Modeling and RPC Language, optimized for fast prototyping – like gRPC, but with a human face
+- [scanamo](https://github.com/scanamo/scanamo): Simpler DynamoDB access for Scala
```

Comment:
Thank you 😚 Also I'm rewriting the core of scanamo to move away from free and embrace effect rotation and environments, which is a super interesting exercise.

Reply:
@regiskuckaertz I'm a happy Scanamo user so thank YOU ;)